### PR TITLE
83 insured token precision changes

### PR DESCRIPTION
**Pull request by shaharyakir**
Based on these calculations:

image

### CODE REVIEW

File: contracts/insured-vesting-v1/InsuredVestingV1.sol [hunk 1]
```diff
@@ -8,9 +8,12 @@ import {Math} from "@openzeppelin/contracts/utils/math/Math.sol";
 contract InsuredVestingV1 is Ownable {
     using SafeERC20 for IERC20;
 
+    uint256 TOKEN_RATE_PRECISION = 1e20;
+
+    // TODO(Audit) comment - rename "PROJECT_TOKEN" (FUNDING_TOKEN for insured)
     IERC20 public immutable USDC;
     IERC20 public immutable XCTD;
-    uint256 public immutable USDC_TO_XCTD_RATE;
+    uint256 public immutable XCTD_TO_USDC_RATE; // TODO(audit) rename to PROJECT_TOKEN_TO_FUNDING_TOKEN_RATE
```
Comment (shaharyakir) [inline contracts/insured-vesting-v1/InsuredVestingV1.sol:16]: intentionally not renaming yet to generic token names, will be modified in a subsequent PR

File: contracts/insured-vesting-v1/InsuredVestingV1.sol [hunk 2]
```diff
@@ -8,9 +8,12 @@ import {Math} from "@openzeppelin/contracts/utils/math/Math.sol";
 contract InsuredVestingV1 is Ownable {
     using SafeERC20 for IERC20;
 
+    uint256 TOKEN_RATE_PRECISION = 1e20;
```
Comment (shaharyakir) [inline contracts/insured-vesting-v1/InsuredVestingV1.sol:11]: intentionally using 1e20 to not confuse with the expected token usage of 1e18

**Comment by shaharyakir** [comment]
resolves #83

File: contracts/insured-vesting-v1/InsuredVestingV1.sol (hunk 2 above)
Comment (Luke-Rogerson) [reply contracts/insured-vesting-v1/InsuredVestingV1.sol:11]: Few lint warnings on line 11

**Review by Luke-Rogerson — APPROVED** [review]

**Pull request merged**